core/imageboard/misc.php

```diff
@@ -24,7 +25,8 @@ function move_upload_to_archive(DataUploadEvent $event): void
 /**
  * Add a directory full of images
  *
- * #return string[]
```

Comment:
In these cases I've been using PHP's native type hints instead of duplicating the same info in the comment, as duplication doesn't seem to add anything - except in the case of arrays, where PHP doesn't support typed arrays, but PHPDoc comments do

Reply:
Thanks for the feedback, I'll keep that in mind